audit-base/src/main/java/com/wultra/core/audit/base/util/JsonUtil.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2021 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wultra.core.audit.base.util;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+/**
+ * JSON serialization utilities.
+ *
+ * @author Roman Strobl, roman.strobl@wultra.com
+ */
+public class JsonUtil {
+
+    private static final Logger logger = LoggerFactory.getLogger(JsonUtil.class);
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
This could maybe use a bit more fine-grained initialization in the constructor, i.e., to enforce the right date format, avoid serializing null value, etc.